README.md

```diff
@@ -9,44 +9,59 @@ request structure.
 We use this test to gain insight into your approach when solving a general problem, and to
 gauge your familiarity with Django and its related dependencies. So if you are unable to complete
 all the requirements due to time constraints or other issues with the local environment, please
-submit your partial work. Consider adding comments or a readme with some next steps you would take
-if you had more time.
+submit your partial work.
 
 While we are a test-driven engineering team, we understand take-home assessments can be
 time-consuming and do not expect unit tests for this project, though if you'd like to write some
 by all means feel free!
 
+**Please add comments or update this readme with some next steps you would take if you had more
```

Comment:
Please comment in your assessment or update this readme with your next steps give you had more time